NEWS

```diff
@@ -11,7 +11,13 @@ CHANGES WITH 257 in spe:
           be updated accordingly. This change has been made to make it harder
           to accidentally delete too many files when using --purge incorrectly.
 
-        Announcements of Future Feature Removals and Incompatible Changes:
+        * The systemd-creds 'cat' verb could previously only read encrypted
+          credentials if they were raw binary data, making it inconsistent
+          with the 'decrypt' verb and and the LoadCredentialEncrypted= service
+          setting, which expect base64-encoded data. This is now fixed,
```

Comment:
I'd always mention the new behavior first, and then describe the previous (wrong) behavior. Also, "potentially breaking existing setups" is redundant.

Reply:
Assuming silence == approval, unresolve if not.